src/app/components/Button/Button.module.css

```diff
@@ -0,0 +1,28 @@
+.submit {
+  background-color: var(--main-font-color);
+  color: var(--secondary-font-color);
+  font: var(--main-font);
```

Comment:
If you use this font for every button, consider adding a base class or changing it in your globals.css.